docs/devops-guide/quickstart.md

```diff
@@ -138,16 +152,18 @@ During installation of Jitsi Meet you can choose between different options:
 
 ### Install Jitsi Meet
 
-_Note_: The installer will check if [Nginx](https://nginx.org/) or [Apache](https://httpd.apache.org/) are present (in that order) and configure a virtual host within the web server it finds to serve Jitsi Meet. If none of the above is found it then defaults to Nginx.
+_Note_: The installer will check if [Nginx](https://nginx.org/) or [Apache](https://httpd.apache.org/) are present (in that order) and configure a virtual host within the web server it finds to serve Jitsi Meet.
 
 If you are already running Nginx on port 443 on the same machine, turnserver configuration will be skipped as it will conflict with your current port 443.
 
 
 ```sh
 # jitsi-meet installation
-sudo apt install jitsi-meet
+sudo apt install --no-install-recommends jitsi-meet
 ```
 
+_Note_: `--no-install-recommends` is necessary for now, because of wrong or missing dependencies, that jitsi-meet would install.
```

Comment:
I'm not aware of any problem here, can you clarify?

Reply:
Some users reported that your packages include incompatible dependencies (for example [here](https://github.com/jitsi/handbook/pull/78#discussion_r522435781)).

Thus these users recommended manual installation of dependencies and using the `--no-install-recommends` flag to prevent from automatic installation of dependencies.

Reply:
That was a while  ago, we test every stable release with a from-scratch installation and haven't seen those anymore. We reverted some of the conflicting changes IIRC.

Reply:
Ok, after the explanation below I added it.